.travis.yml

```diff
@@ -10,6 +10,8 @@ matrix:
           env: KERAS_BACKEND=tensorflow TEST_MODE=INTEGRATION_TESTS PIL=Pil
         - python: 3.6
           env: KERAS_BACKEND=tensorflow TEST_MODE=PEP8_DOC PIL=Pillow
+        - python: 3.6
+          env: KERAS_BACKEND=tensorflow TEST_MODE=API
```

Comment:
Is there a way to just add a label to the PR rather than just making the whole travis build fail?